subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -241,6 +243,12 @@ struct node_rx_ftr {
 #endif /* CONFIG_BT_HCI_MESH_EXT */
 };
 
+/* Meta-information for isochronous PDUs in node_rx_hdr */
+struct node_rx_iso_meta {
+	uint64_t payload_number; /* 39-bit cisPayloadNumber */
```

Comment:
```suggestion
	uint64_t payload_number :39  ; /* cisPayloadNumber */
```

Reply:
Sorry for overlooking that one, thanks for pointing it out.